internal/driver/driver.go

```diff
@@ -246,6 +246,11 @@ func aggregate(prof *profile.Profile, cfg config) error {
 		function = true
 		filename = true
 		linenumber = true
+	case "columns":
```

Comment:
I think this shouldn't be in the granularity switch anymore and should be handled as option instead?

Reply:
Done.